software/configs/nctrl.service

```diff
@@ -2,8 +2,10 @@
 Description=The AXIOM nctrl daemon to expose a fuse api
 
 [Service]
-ExecStart=/usr/axiom/bin/nctrl --mountpoint /axiom-api/ $(realpath /etc/axiom-yml)
+ExecStart=bash -c "/usr/axiom/bin/nctrl --mountpoint /axiom-api/ $(realpath /etc/axiom-yml)"
 StandardOutput=journal
+# this is due to some random crashes and not fully convincing error handling in nctrl
```

Comment:
do you have some backtraces / a way to replicate for these crashes?

Reply:
no, sadly not :(